### PR TITLE
Add community meeting calendar event

### LINKED ISSUE #16: Add community meeting calendar event to the community section
## Please describe what you would like to see in this project

Add community meeting calendar event to the community section

### PR DESCRIPTION
**Pull request by sandrobonazzola**
## Changes introduced with this PR

Add community meeting calendar event.

Fixes #16 

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -62,3 +62,4 @@ You can find our general community health files like our code of conduct and con
 
 If you would like to contribute, check out the issues in the individual repositories and our [project boards](https://github.com/orgs/arcalot/projects) where we organize our work.
 
+If you want to participate to our bi-weekly meeting you can add out recurring meeting to your calendar: [Arcalot Project Community meeting (iCal)](meetings/arcalot-community-biweekly.ics)
```
Comment (portante) [inline docs/index.md:65]: s/out/our

File: docs/arcaflow/creating-plugins/python.md [hunk 2]
```diff
@@ -362,7 +362,7 @@ You can add metadata to your schema by using the `field()` parameter for datacla
 ```python
 @dataclasses.dataclass
 class MyClass:
-    param: str = field(metadata={"id": "my-param", "name":"Parameter 1", "description": "This is a parameter"})
+    param: str = dataclasses.field(metadata={"id": "my-param", "name":"Parameter 1", "description": "This is a parameter"})
```
Comment (portante) [inline docs/arcaflow/creating-plugins/python.md:365]: Is this intended to be part of this PR?

**Review by portante — CHANGES_REQUESTED** [review]
Looks good!

File: docs/arcaflow/creating-plugins/python.md (hunk 2 above)
Comment (sandrobonazzola) [reply docs/arcaflow/creating-plugins/python.md:365]: no, got in by mistake, checking

File: docs/index.md (hunk 1 above)
Comment (sandrobonazzola) [reply docs/index.md]: thanks, fixed

File: docs/index.md [hunk 3]
```diff
@@ -62,3 +62,4 @@ You can find our general community health files like our code of conduct and con
 
 If you would like to contribute, check out the issues in the individual repositories and our [project boards](https://github.com/orgs/arcalot/projects) where we organize our work.
 
+If you want to participate to our bi-weekly meeting you can add our recurring meeting to your calendar: [Arcalot Project Community meeting (iCal)](meetings/arcalot-community-biweekly.ics)
```
Comment (portante) [inline docs/index.md:65]: Sorry, one more to consider, `s/to participate to our bi-weekly/to participate in our bi-weekly/`

**Review by portante — APPROVED** [review]
One minor typ-o I missed, but looks good otherwise.

**Review by sanjacodes — APPROVED** [review]

File: docs/index.md (hunk 3 above)
Comment (sandrobonazzola) [reply docs/index.md]: thanks, fixed

**Review by portante — COMMENTED** [review]
Great!

**Pull request merged**